plugins/autowarmth.koplugin/_meta.lua

```diff
@@ -2,5 +2,5 @@ local _ = require("gettext")
 return {
     name = "autowarmth",
     fullname = require("device"):hasNaturalLight() and _("Auto warmth and night mode") or _("Auto night mode"),
-    description = _([[This plugin allows to set the frontlight warmth automagically.]]),
+    description = require("device"):hasNaturalLight() and _([[This plugin allows to set the frontlight warmth and night mode automagically.]]) or _([[This plugin allows to set night mode automagically.]]),
```

Comment:
It's not really set, is it.
```suggestion
    description = require("device"):hasNaturalLight() and _([[This plugin allows to set the frontlight warmth and night mode automagically.]]) or _([[This plugin allows to enable night mode automagically.]]),
```